src/createServer.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { resetUsers } = require('./services/user.service');
+const { userRouter } = require('./routes/user.route');
+const { resetExpenses } = require('./services/expense.service');
+const { expenseRouter } = require('./routes/expense.route');
 
 function createServer() {
   // Use express to create a server
   // Add a routes to the server
   // Return the server (express app)
+  const app = express();
+
+  resetUsers();
+  resetExpenses();
+
+  app.use('/users', express.json(), userRouter);
```

Comment:
`express.json()` can be used for all routes: `app.use(express.json());`. Make sure to place it before routes.
After this changed is applied, it can be removed from the specific routes.